ui/src/app/components/html-viewer.tsx

```diff
@@ -51,16 +52,19 @@ export const HtmlViewer = withWindowSize()(
         body.prepend(openLinksInNewTab);
         body.appendChild(endOfPage);
 
+        const elements = iframeDocument.querySelectorAll('.MsoNormal');
+        const last = elements[Object.keys(elements).length - 1];
+
         const observer = new IntersectionObserver(
-          // The callback receives a list of entries - since we only have one intersection entry (threshold: 1.0)
+          // The callback receives a list of entries - since we only have one intersection entry (threshold: 0.1)
           // we can destructure the first item of the array and determine whether it is intersecting or not
           ([{ isIntersecting }]) =>
             isIntersecting &&
             !this.state.hasReadEntireDoc &&
             this.setState({ hasReadEntireDoc: true }),
-          { root: null, threshold: 1.0 }
+          { root: null, threshold: 0.1 }
```

Comment:
threshold = "how much of the object must be visible".  Changed from "all" to "a tiny piece"

Also changed *which* object was visible.  This one takes up the whole width of the screen, rather than a tiny invisible div on the far left - easily out of the scrolling area on small screens